api/main.py

```diff
@@ -31,6 +32,7 @@
     backend=TokenAuthentication(token=settings.auth_token),
     on_error=on_auth_error,
 )
+app.include_router(api_router)
```

Comment:
Since, we are using api versioning, wouldn't it be better to make a file maybe called `api.py` in the version folder i.e. `v1` right now and just include all the routers in that and then include that router here? Otherwise we would have to change it when we change the api version.

main.py
```py
app.include_router(v1_api_router)
```

api.py
```py
v1_api_router = APIRouter()
v1_api_router.include_router(auth.router, prefix="/auth")
v1_api_router.include_router(users.router, prefix="/users")
```

Reply:
There's currently no plan of using strict API versioning, so I'm not sure if it's worth doing this at the moment. Via the comment in `api/v1/__init__.py`:

> There are currently no plan to use
> a strictly versioned API design, as this API is currently
> tightly coupled with a single client application.

Reply:
Hmm, if that's case I don't see the reason to name it `v1` it is rather confusing as it says there is API versioning on one side and on the other it says there are no plans.

We can name it `api` or `endpoints` maybe, and in case there are plans to a versioned API design we could shift?

Reply:
Sure, I'll do a separate PR, because this PR didn't touch the package structure, so it's kind of out of scope here.

Reply:
See https://github.com/python-discord/api/pull/15